<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>R, Reucher</author>
 <class>FileEditWidget</class>
 <widget class="QWidget" name="FileEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonBrowse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButtonClear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/broom.png</normaloff>:/data/img/broom.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>lineEditFile</tabstop>
  <tabstop>toolButtonBrowse</tabstop>
  <tabstop>toolButtonClear</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>lineEditFile</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
